src/apps/search-header/search-header.tsx

```diff
@@ -63,6 +88,13 @@ const SearchHeader: React.FC = () => {
       textData.push(item);
     });
     orderedData = textData.concat(materialData);
+
+    if (nonWorkSuggestion) {
+      autosuggestCategoryList.forEach(() => {
+        categoryData.push(nonWorkSuggestion as Suggestion);
```

Comment:
Mmm I think it's not that much about the undefined case. We made our own type called Suggestion - see pic:
![image](https://user-images.githubusercontent.com/28546954/187677505-45cf1938-a4e9-402b-a3f3-5a8cfe58359a.png)
Because we already check that nonWorkSuggestion exists and has a value on line 92. I'll leave it as is.